src/ffpuppet/profile.py

```diff
@@ -0,0 +1,265 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""ffpuppet profile manager"""
+
+from json import load as json_load
+from logging import getLogger
+from pathlib import Path
+from shutil import copyfile, copytree, rmtree
+from subprocess import STDOUT, CalledProcessError, check_output
+from tempfile import mkdtemp
+from time import time
+from typing import Any, Dict, List, Optional
+from xml.etree import ElementTree
+
+from .helpers import onerror
+
+LOG = getLogger(__name__)
+
+__author__ = "Tyson Smith"
+__all__ = ("Profile",)
+
+
+def certutil_available() -> bool:
+    """Check if NSS certutil is available.
+
+    Args:
+        None
+
+    Returns:
+        True if certutil is available otherwise False.
+    """
+    try:
+        check_output(["certutil"], stderr=STDOUT)
```

Comment:
why not try using the certutil that comes with the target?

Reply:
`certutil` isn't in the target, it's in the common tests download, which usually won't be present. my mistake.